app/app.h

```diff
@@ -1,7 +1,49 @@
 #pragma once
 #include <string>
+#include <vector>
+#include <cctype>
 
-namespace vsite::oop::v3 
-{
+namespace vsite::oop::v3
+{	
+		struct student
+		{
+			std::string name;
+			int grade;
+
+			student(const std::string& name = "pero", int grade = 2);
+		};
+
+		static bool has_grade_check(const student& s, int grade);
```

Comment:
Zašto su globalne funkcije `static`? Zašto su uopće deklarirane u headeru kad ih koristite samo u app.cpp?

Reply:
ispraviti ću